Formula/w/whatweb.rb

```diff
@@ -0,0 +1,36 @@
+class Whatweb < Formula
+  desc "Web scanner. Identify the technology stack that powers a website"
+  homepage "https://morningstarsecurity.com/research/whatweb"
+  url "https://github.com/urbanadventurer/WhatWeb/archive/refs/tags/v0.5.5.tar.gz"
+  sha256 "96dedb6a377184fb8f5fd3f2a81c26ff8c92c4dc1503ce409793a1e7ab23695d"
+  license "GPL-2.0-only"
+  head "https://github.com/urbanadventurer/WhatWeb.git", branch: "master"
+
+  uses_from_macos "ruby" => :build, since: :ventura
+
+  def install
+    ENV["GEM_HOME"] = buildpath/"gem_home"
+    system "gem", "install", "bundler"
+    ENV.prepend_path "PATH", buildpath/"gem_home/bin"
+    system "bundle", "config", "--local", "#{libexec}/whatweb"
+    system "bundle", "update"
```

Comment:
```suggestion
```

We want to use the versions in `Gemfile.lock` here.

Reply:
Thanks @MikeMcQuaid ok i will remove this, i put it there as a last ditch effort for a job i'm having trouble with.

This [job](https://github.com/Homebrew/homebrew-core/actions/runs/6146173885/job/16675114448#step:3:50) keeps failing due to a missing gem `addressable`, which doesn't make sense to me since it's included in the Gemfile. I also tried testing the forumula on my mac computer and a Debian 10 system and they don't have this problem. I don't know if this is a CI problem or a problem for the venfor